db/migrate/20180130165200_drop_webmks.rb

```diff
@@ -0,0 +1,7 @@
+class DropWebmks < ActiveRecord::Migration[5.0]
```

Comment:
This is ok to have here, as long as

- it must be in manageiq-schema and identically named and with identical comments
- we do not expect customers to have to execute this schema change on a z-stream update.